quipucords/fingerprinter/runner.py

```diff
@@ -1464,6 +1465,18 @@ def _process_ansible_fact(self, source, fact):
         )
         return fingerprint
 
+    def _process_acs_fact(self, source, fact):
+        """Generate fingerprints for ACS facts."""
+        fingerprint = {
+            META_DATA_KEY: {},
+            ENTITLEMENTS_KEY: [],
+            PRODUCTS_KEY: [],
+        }
+        self._add_fact_to_fingerprint(
+            source, "N/A", fact, "name", fingerprint, fact_value="collected-from-acs"
```

Comment:
I don't see that we're using the `"N/A"` anywhere else in our code other than the new test fixture you added. Is there any special significance to that value here, aside from the literal "not applicable" meaning? Are we using `"N/A"` simply because a not-`None`, not-empty string value is required for `_add_fact_to_fingerprint`'s `raw_fact_key` parameter?

Reply:
@infinitewarp This is the first time the fingerprint doesn't have a matching raw fact. We're going with "N/A" to fill in for the missing raw fact name.

Reply:
Okay, thanks for the explanation! If this becomes a more-than-just-once pattern, we should consider changing the way we handle `raw_fact_key` or at least define a shared constant to use instead of propagating strings that might not always look the same. For now, though, this is probably fine.

(I'm leaving this comment thread open in case you have any additional thoughts. If not, please feel free to resolve it!)